admin/class-premium-upsell-admin-block.php

```diff
@@ -58,8 +58,16 @@ public function render() {
 
 		/* translators: %s expands to "Yoast SEO Premium". */
 		$dismiss_msg = sprintf( __( 'Dismiss %s upgrade motivation', 'wordpress-seo' ), 'Yoast SEO Premium' );
-		/* translators: %s expands to "Yoast SEO Premium". */
-		$upgrade_msg = sprintf( __( 'Find out why you should upgrade to %s &raquo;', 'wordpress-seo' ), 'Yoast SEO Premium' );
+
+		/* translators: %s expands to Yoast SEO Premium */
+		$button_text = sprintf( __( 'Get %s', 'wordpress-seo' ), 'Yoast SEO Premium' );
+
+		$upgrade_button = sprintf(
+			'<a id="wpseo-%1$s-popup-button" class="yoast-button-upsell" href="%2$s" target="_blank" rel="noreferrer noopener">%3$s</a>',
+			$this->identifier,
+			$url,
+			$button_text
+		);
 
 		echo '<div class="' . esc_attr( $class ) . '">';
 		printf(
```

Comment:
The "X" button in the line below looks a bit different in the provided design:
- the X character should be changed
- the button style should be updated